site/_posts/2019-10-01-implementing-fusionauth-python.md

```diff
@@ -65,11 +65,11 @@ sudo service fusionauth-search start
 sudo service fusionauth-app start
 ```
 
-With installation complete and services started, let's open up a web browser and navigate to [http://127.0.0.1:9011](http://127.0.0.1:9011) or the web address pointing to your FusionAuth installation. If you're connected over SSH, be sure to first tunnel in to port 9011 so that we can access the server's localhost.
+With installation complete and services started, let's open up a web browser and navigate to [http://127.0.0.1:9011](http://127.0.0.1:9011) or the web address pointing to your FusionAuth installation. If you're connected over SSH, be sure to first tunnel into port 9011 so that we can access the server's localhost.
```

Comment:
In the past year, we've moved away from hyperlinking to localhost.

So you could change that to `\http://localhost:9011` which I believe doesn't hyperlink.

Reply:
I think this was primarily because people would click on them and get dead links, or I think we've even seen our Google search index complain about dead links for this type of thing.